### PR TITLE
NavLinks only. SubheaderLinks removed.

### PR DESCRIPTION
**Pull request by mwilkins74**

### CODE REVIEW

File: src/components/molecules/NavList.tsx [hunk 1]
```diff
@@ -31,34 +26,59 @@ export function NavList() {
...
+    'Animation',
+    'Documentary',
+    'Director of Photography',
+    'Director & Producer',
+    'Series',
+  ];
+
+  const secondNavLinks = combinedLinks
+    .filter(item => secondNavLinkOrder.includes(item.label))
+    .sort((a, b) => secondNavLinkOrder.indexOf(a.label) - secondNavLinkOrder.indexOf(b.label));
```
Comment (ChuckBTaylor) [inline src/components/molecules/NavList.tsx:51]: 🔧 This feels like a lot of work to get a list of ordered `NavLinks`

Something like:

```typescript
const orderedLinks: (Genre | Role)[] = ['animation', 'documentary', 'director_of_photography', 'director_producer', 'series']
//...
function NavList(){
  // ...
  const secondNavLinks = orderedLinks.map(linkEnum => (
    <NavLinks
      key={`burger-link-to-${linkEnum}`}
      route={enumMappings[linkEnum].url
      label={enumMappings[linkEnum].label}
      onClick={() => navigate(enumMappings[linkEnum].url)}
    />
  )
  ...
}
```

File: src/components/molecules/NavList.tsx [hunk 2]
```diff
@@ -31,34 +26,59 @@ export function NavList() {
...
-      ))}
-    </List>
+      <List
+        sx={{ width: '100%', height: '100%', maxWidth: 360, ...NavStyle }}
+        component="nav"
+      >
+        {secondNavLinks.map((item, index) => (
+          <NavLinks
+            key={index}
+            route={item.path}
```
Comment (ChuckBTaylor) [inline src/components/molecules/NavList.tsx:76]: ❓ Why give a `route` and an `onClick` that navigates?

Comment (mwilkins74) [reply src/components/molecules/NavList.tsx:76]: `route` is not necessary and I just didn't think about it when I sent up this PR. 
We (I) can just utilize `useNavigate() here.

File: src/components/molecules/NavList.tsx (hunk 1 above)
Comment (mwilkins74) [reply src/components/molecules/NavList.tsx:51]: Ok, it did seem like a lot when I wrote it, but was also the solution I could get to today. Couldn't figure out a way to make this more concise. 

Your rewrite is great!

I'll figure this out tomorrow.

File: src/components/molecules/NavList.tsx [hunk 3]
```diff
@@ -1,64 +1,66 @@
...
-  filter: string
+  filter?: string
   path: string
 }
-type SubheaderLinksProps = {
-  items: readonly (Genre | Role)[]
-  navigate: (route: string) => void
-}
+
+const orderedLinks: ((typeof genres)[number] | (typeof roles)[number])[] = [
```
Comment (ChuckBTaylor) [inline src/components/molecules/NavList.tsx:14]: ❓ Why not `: (Genre | Role)[]`

File: src/components/molecules/NavList.tsx [hunk 4]
```diff
@@ -1,64 +1,66 @@
...
-  ): ListButtonProps[] =>
-    items.map(item => ({
-      label: enumMappings[item].label,
-      filter: item,
-      path: enumMappings[item].url, 
-    }));
+  const combinedLinks: ListButtonProps[] = navLinks.map(item => ({
+    label: item.label,
+    path: item.route,
+  }))
```
Comment (ChuckBTaylor) [inline src/components/molecules/NavList.tsx:28]: 🔧 Similar question here. Why convert the `navlinks` to `ListButtonProps` here and then just convert them to `NavLinks` later? Why not just go straight to `NavLinks`?

File: src/components/molecules/NavList.tsx (hunk 3 above)
Comment (mwilkins74) [reply src/components/molecules/NavList.tsx:14]: Cause not my smartest code...

File: src/components/molecules/NavList.tsx (hunk 4 above)
Comment (mwilkins74) [reply src/components/molecules/NavList.tsx:28]: You're right. 
Hate when I don't catch stuff like this.

**Review by kmdunn5 — APPROVED** [review]

**Pull request merged**